### DISCUSSION TITLE
Sunbird RC eLocker related - User Persona Action #3 - Most useful addition

### DISCUSSION THREAD

**Discussion by jagadishbabu** [Sunbird-RC/community#349] [Ideas & Enhancements]
Sunbird RC eLocker - Userpersona 
**Accepting Party/3rd Party: **
(WIP)I would like to access documents of my customer easily, so that I can review and process the next steps of my customer’s claim…

eLocker today are used extensively to store documents. I see that User action #3 which deals with access to documents upon consent electronically as WIP. I would like to stress upon the need to work on this thread and have this flow as part of reference implementation. Without this important addition, elockers tend to be mere document storing mechanisms without ever being used for authenticated use thus reducing the time needed for claim verifications. I see automating claim verifications as important value add to users of elockers.

**Comment by parthlawate** [comment]
@jagadishbabu this is definitely something we should pickup . A detailed thread on this has been started on this here https://github.com/orgs/Sunbird-RC/discussions/348 .. 

The discussion can be continued there.
